_includes/integrations/react-tech-ref.md

```diff
@@ -74,16 +127,18 @@ To register for a {{site.cloudname}} API key, visit the [sign-up page]({{site.ac
 ```
 
 #### `cloudChannel`
+Changes the {{site.productname}} build used for the editor to either a specific version or a channel indicating a stability level.
 
 **Default value:** `{{site.productmajorversion}}-stable`
 
-**Possible values:**  `{{site.productmajorversion}}-stable`, `{{site.productmajorversion}}-testing`, `{{site.productmajorversion}}-dev`
+**Possible values:**  `{{site.productmajorversion}}-stable`, `{{site.productmajorversion}}-testing`, `{{site.productmajorversion}}-dev`, `{{site.productminorversion}}.0`
 
 Changes the {{site.productname}} build used for the editor to one of the following {{site.cloudname}} channels:
 
 - `{{site.productmajorversion}}-stable` (**Default**): The current enterprise release of {{site.productname}}.
 - `{{site.productmajorversion}}-testing`: The current release candidate for the next enterprise release of {{site.productname}}.
 - `{{site.productmajorversion}}-dev`: The nightly-build version of {{site.productname}}.
+- A version number such as `{{site.productminorversion}}.0`: The specific enterprise release version of {{site.productname}}.
```

Comment:
As mentioned above, we really should avoid using/recommending  a specific patch release and should only do up to the minor version if we want to include that.

Reply:
done